### PR TITLE
Support mu-plugins-ext in mu-plugins image

### PR DESCRIPTION
**Pull request by pschoffer**
# Description
We are working towards moving versioned Jetpack folders outside of mu-plugins into this mu-plugins-ext repo.

That way we can automate updates of those dependencies and combine them during deployment.

### CODE REVIEW

File: mu-plugins/run.sh [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 if [ ! -d /shared/.git ]; then
   rsync -a --delete --delete-delay /mu-plugins/ /shared/
+  rsync -r --exclude-from="/mu-plugins-ext/.dockerignore" /mu-plugins-ext/* /shared
```
Comment (rinatkhaziev) [inline mu-plugins/run.sh:7]: Should this also have a`--delete` flag? 
```suggestion
  rsync -r --delete --exclude-from="/mu-plugins-ext/.dockerignore" /mu-plugins-ext/* /shared
```

**Review by rinatkhaziev — COMMENTED** [review]
Thanks, @pschoffer I left a note.

File: mu-plugins/autoupdate.sh [hunk 2]
```diff
@@ -9,6 +9,7 @@ while true; do
 	if [ "$head" != "$develop" ]; then
 		git reset --hard origin/develop
 		git submodule update
+		rsync -r --exclude-from="/mu-plugins-ext/.dockerignore" /mu-plugins-ext/* ./
```
Comment (pschoffer) [inline mu-plugins/autoupdate.sh:12]:
```suggestion
		rsync -r  --delete --exclude-from="/mu-plugins-ext/.dockerignore" /mu-plugins-ext/* ./
```

**Comment by rinatkhaziev** [comment]
awesome

**Review by rinatkhaziev — APPROVED** [review]

**Pull request merged**